patches/net/minecraft/world/level/block/entity/AbstractFurnaceBlockEntity.java.patch

```diff
@@ -31,7 +31,7 @@
 +        this.recipeType = p_154994_;
      }
  
-+    /**@deprecated Forge: get burn times by calling CommonHooks#getBurnTime(ItemStack)*/ @Deprecated
++    /**@deprecated Neo: get burn times by calling {@link net.neoforged.neoforge.common.extensions.IItemStackExtension#getBurnTime(RecipeType)}*/ @Deprecated
```

Comment:
This patch should be updated to look "normal" - it doesn't need to be a one-liner

```java
/**
 * @deprecated Neo: Get fuel burn times by calling {@link net.neoforged.neoforge.common.extensions.IItemStackExtension#getBurnTime(RecipeType)}
 */ 
 @Deprecated